Bio/Cluster/__init__.py

```diff
@@ -408,14 +408,14 @@ def distancematrix(self, transpose=0, dist='e'):
         The distance matrix is returned as a list of 1D arrays containing the
         distance matrix between the gene expression data. The number of columns
         in each row is equal to the row number. Hence, the first row has zero
-        elements. An example of the return value is::
```

Comment:
That should be a double colon, its how you tell RST there is a literal block next:
http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#literal-blocks